db/people.yml

```diff
@@ -1291,6 +1508,27 @@ jcalamari:
     - level: beginner
       category: Specialized Software
       name: SolidWorks
+jchung:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
+      institution: fix.me
+      degree: fix.me
```

Comment:
I think this was a student of Mike Thorpe at MSU and he visited me again later after he left